easybuild/toolchains/compiler/inteliccifort.py

```diff
@@ -104,6 +104,9 @@ def _set_compiler_vars(self):
             raise EasyBuildError("_set_compiler_vars: mismatch between icc version %s and ifort version %s",
                                  icc_version, ifort_version)
 
+        # reset LIB_MULTITHREAD every time,
+        # to avoid problems when multiple Intel compilers versions are used in a single session
+        self.LIB_MULTITHREAD = ['iomp5', 'pthread']  # iomp5 is OpenMP related
```

Comment:
should, we had a very good reason at some point for this, and I'm not keen on breaking stuff that works just to remove a handful of characters ;)